evaluation.py

```diff
@@ -237,7 +237,97 @@ def predict_keypoints(self,generator, location):
                 image_ids.append(metadata['src_set_image_id'])
                 j+=1
         self.write_to_json_file(list_of_predictions, location)
-        return image_ids
+        return image_ids, list_of_predictions
```

Comment:
This additional return type may affect the OKS script, in `run_evaluation.py` line 147

Reply:
I believe it should only take the first return value, but I will double check!

Reply:
Good catch I will fix that 👍